src/managers/GuildManager.js

```diff
@@ -211,6 +211,25 @@ class GuildManager extends BaseManager {
         }, reject),
     );
   }
+
+  /**
+   * Obtains a guild from Discord, or the guild cache if it's already available.
+   * @param {Snowflake} id ID of the guild
+   * @param {boolean} [cache=true] Whether to cache the new guild object if it isn't already
+   * @returns {Promise<Guild>}
+   * @example
+   * // Fetch a guild by its id
+   * client.guilds.fetch('222078108977594368')
+   *   .then(guild => console.log(guild.name))
+   *   .catch(console.error);
+   */
+  async fetch(id, cache = true) {
+    const existing = this.cache.get(id);
+    if (existing) return existing;
+
+    const data = await this.client.api.guilds(id).get();
+    return this.add(data, null, cache);
```

Comment:
```suggestion
    return this.add(data, cache);
```
https://github.com/discordjs/discord.js/blob/2583ad5da7db5ab92dfa357909f9c713db0a2981/src/managers/BaseManager.js#L44